desktop/core/src/desktop/manage_entry.py

```diff
@@ -85,16 +85,19 @@ def entry():
 
   if len(sys.argv) > 1:
     subcommand = sys.argv[1]
-  parser = CommandParser(None, usage="%(prog)s subcommand [options] [args]", add_help=False)
+  if sys.version_info[0] < 3:
+    parser = CommandParser(None, usage="%(prog)s subcommand [options] [args]", add_help=False)
+  else:
+    parser = CommandParser(usage="%(prog)s subcommand [options] [args]", add_help=False)
```

Comment:
Any way to avoid the duplications?

e.g.
if py2:
  args = [None]
else:
  args = []
CommandParser(*args, **kwargs)

and even maybe
CommandParser(*args, usage="%(prog)s subcommand [options] [args]", add_help=False)